src/popup/views/authorize/authorize.jsx

```diff
@@ -20,9 +21,11 @@ class AuthorizeView extends React.Component {
   async componentDidMount () {
     try {
       const pricing = await getTxPricing(this.props.current);
-
       this.props.dispatch(updatePricing(pricing.map(i => i.result)));
 
+      const smartLockerState = await getSmartLockerState();
```

Comment:
I don't think we should break components paradigm. This disrupts component hierarchy and it's invitation for spaghetti code. 

I don't think we really need to update locker state here. `<Locker/>` component can still manage it by itself.

This made me thinking that sooner or later we should review state / data flow through rest of the components.

Reply:
Done